.github/workflows/ci-backend-cql.yml

```diff
@@ -102,9 +115,46 @@ jobs:
           java-version: 1.8
       - run: mvn clean install --projects janusgraph-${{ matrix.module }} ${{ env.BUILD_MAVEN_OPTS }} ${{ matrix.install-args }}
       - run: mvn verify --projects janusgraph-${{ matrix.module }} ${{ env.VERIFY_MAVEN_OPTS }} ${{ matrix.args }}
-      - name: cql 3.0.18
-        if: "github.event_name == 'push' && contains(github.event.head_commit.message, '[cql-tests]') || github.event_name == 'pull_request' && contains(github.event.pull_request.title, '[cql-tests]')"
-        run: mvn verify --projects janusgraph-${{ matrix.module }} -Dcassandra.docker.version=3.0.18' ${{ env.VERIFY_MAVEN_OPTS }} ${{ matrix.args }}
       - uses: codecov/codecov-action@v1
         with:
           name: codecov-cql-${{ matrix.name }}
+
+  full-tests:
+    runs-on: ubuntu-20.04
+    if: "github.event_name == 'push' && contains(github.event.head_commit.message, '[cql-tests]') || github.event_name == 'pull_request' && contains(github.event.pull_request.title, '[cql-tests]')"
+    needs: build-all
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - module: cql
```

Comment:
@li-boxuan Do we want to name these also?

Reply:
Naming is only required for codecov report. Since we don't enable codecov for full-test, we don't need to name them here.